components/dashboard/src/projects/Prebuilds.tsx

```diff
@@ -159,7 +153,8 @@ export default function () {
                 <div className="py-3 pl-3">
                     <DropDown prefix="Prebuild Status: " contextMenuWidth="w-32" entries={statusFilterEntries()} />
                 </div>
-                <button disabled={!project} onClick={() => triggerPrebuild(null)} className="ml-2">Trigger Prebuild</button>
+                {(!!project && prebuilds.length === 0) &&
+                    <button onClick={() => triggerPrebuild(null)} className="ml-2">Run Prebuild</button>}
```

Comment:
This could be improved as an "empty state" for the Prebuilds page (but maybe after we rework it to be per-branch).

Reply:
True, an empty state for the Prebuilds page (with an illustration, and maybe this infamous `Run Prebuild` button), or for the upcoming "Prebuilds of branch 'X'" page (as described in https://github.com/gitpod-io/gitpod/issues/5791#issuecomment-931434754), could make a lot of sense. 💯

This would also solve the problem that it is impossible to `Run Prebuild` when you already have a (successful) Prebuild for the same branch (because as soon as you have a Prebuild, the empty state including the button will disappear).

Currently, it's a problem to show `Run Prebuild` when you already have a successful Prebuild for the same branch (because clicking on `Run Prebuild` will make the backend respond with "ah, this already exists, job done!" and the user sees no new Prebuild being triggered).

I see the following solutions:
1. Make the `Run Prebuild` button disappear when there are Prebuilds (it's useful when there are no Prebuilds, but confusing & sometimes ineffective when there are Prebuilds) -- implemented here
2. Make the `Run Prebuild` always have a visual effect (i.e. if a new Prebuild can be triggered, the new Prebuild appears in the list, but if there is already a successful Prebuild for the current branch, we could simply highlight it e.g. like a hover state, in order to show "done! this Prebuild is already exactly what you want")